apps/projection/src/app/directive/list-item-template.directive.ts

```diff
@@ -0,0 +1,9 @@
+import {Directive} from "@angular/core";
+
+@Directive({
+  standalone: true,
+  selector: '[list-item-template]'
+})
+export class ListItemTemplateDirective {
+
+}
```

Comment:
very nice 👍 

And you can go one step further and add strong typing to your templateContext. But I will do another challenge in this topic.